wger/exercises/views/muscles.py

```diff
@@ -36,10 +36,14 @@ class MuscleListView(ListView):
     Overview of all muscles and their exercises
     '''
     model = Muscle
-    queryset = Muscle.objects.all().order_by('-is_front', 'name'),
     context_object_name = 'muscle_list'
     template_name = 'muscles/overview.html'
 
+    def get_queryset(self):
```

Comment:
LGTM, wanted to know though why the if statement is required, running on my mac I have noted it cannot run without it, so its important, may be you share the knowledge...